jwst/wfs_combine/wfs_combine.py

```diff
@@ -138,13 +138,16 @@ def create_aligned_2(self):
             # 1. Create smoothed image of input SCI data of image #1
             # 1a. create image to smooth by first setting bad DQ pixels equal
             #     to mean of good pixels
-            data_1 = self.input_1.data.copy()
+            data_1 = self.input_1.data.astype(np.float64)
             wh_bad_1 = np.where(self.input_1.dq != 0)
             wh_good_1 = np.where(self.input_1.dq == 0)
-            data_1[wh_bad_1] = self.input_1.data[wh_good_1].mean()
+            data_1[wh_bad_1] = data_1[wh_good_1].mean()
 
             # 1b. Create smoothed image by smoothing this 'repaired' image
-            s_data_1 = smooth_image(data_1, BLUR_SIZE)
+            g = gauss_kern(BLUR_SIZE, sizey=None)
+            s_data_1 = scipy.signal.convolve(data_1, g, mode='valid')
```

Comment:
Importing `scipy` and then calling `scipy.signal.convolve` does not work for me, using `scipy` version 1.3.1.  This works, though:

    from scipy import signal
    stuff = signal.convolve(other_stuff)

Reply:
I did test this with `scipy 1.1`.  Let me figure out what's going on with 1.3.1.

Reply:
So the regression tests that were failing pass fine with this PR and `scipy==1.3.1`.  What failure were you seeing?

```
$ pytest --bigdata jwst/tests_nightly/general/ -k test_wfs_combine2
========================================================= test session starts ==========================================================
platform darwin -- Python 3.7.5, pytest-5.3.2, py-1.8.1, pluggy-0.13.1
rootdir: /Users/jdavies/dev/jwst, inifile: setup.cfg
plugins: requests-mock-1.7.0, hypothesis-5.1.1, openfiles-0.4.0, xdist-1.31.0, ci-watson-0.4, asdf-2.5.1, forked-1.1.3, doctestplus-0.5.0, cov-2.8.1
collected 220 items / 219 deselected / 1 selected                                                                                      

jwst/tests_nightly/general/nircam/test_nircam_steps_single.py .                                                                  [100%]
============================================ 1 passed, 219 deselected, 5 warnings in 29.98s ============================================
$ pip freeze | grep scipy
scipy==1.3.1
$
```

Reply:
Here is an example:

    import scipy
    help(scipy.signal.convolve)
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
    AttributeError: module 'scipy' has no attribute 'signal'
    scipy.__version__
    '1.3.1'

Reply:
I can reproduce that.  But the code clearly works in the tests.  Hmm.

It seems to be path-dependent.  Is this because `scipy.signal` is a C library and not a module?  Seems to be a namespace import in `signal/__init__.py` from a C library.  Hmm.

I will change it.